New article to spice up the appearing-to-be-dead blog. (#7)
wlogblog/posts: submissions/golden_handcuffs.md

---

> title: "Compensation Manipulation"

Reviewer: ```suggestion
title: "compensation manipulation"
```

lower casing just to fit with style of other H1 titles

date: 2024-04-04T21:08:37-07:00
draft: false
keywords:
  - FAANG
  - golden handcuffs
  - big tech
  - compensation
  - tech jobs
tags:

>   - FAANG

Reviewer: ```suggestion
  - FAANG
  - big tech
  - compensation
  - tech jobs
  - thoughts
```

adding these to tags will enable quick navigation to other similarly tagged future posts

> author: "@the.oneandonlyamericancontributor"

Reviewer: lol

# author_external_url : "https://github.com/wlogblog/"
robots: index,follow
---

Manipulation of all kinds is the norm at big corporations. Corporations are incredible at what they do because they have had ample time to perfect the practice. And if that practice is in manipulating employees, you bet ya, they have perfected it too.

> ### Manipulator's Handbook
> 
> The steps to success for any master manipulator are as follows:
> - Make the target feel like they are in control and the decisions they make are their own;
> - Pull levers that you know only you have the power to leverage;
> - Create a sense of urgency to accept your terms;
> - Create obstacles to exploring alternatives to create a sense of superiority of one choice over others;
> - Strategically place reminders of impermanence & instability to suggest how lucky the target is to be where they are and how grateful they should be;
> - Exploit imposter syndrome, market volatility, and any and all world state conditions in your favor.
> 
> The result? Your target will do exactly what you want, when you want, will apologize profusely for ever thinking you were not God Almighty, and will thank you for the opportunity to slave for you for another couple of years.
> 
> Pretty good.

Reviewer: ```suggestion
> ### :book: Manipulator's Handbook
>
> #### *steps to success for a master manipulator*
> - *Make the target feel like they are in control and the decisions they make are their own*
> - *Pull levers that you know only you have the power to leverage*
> - *Create a sense of urgency to accept your terms*
> - *Create obstacles to exploring alternatives to create a sense of superiority of one choice over others*
> - *Strategically place reminders of impermanence & instability to suggest how lucky the target is to be where they are and how grateful they should be*
> - *Exploit imposter syndrome, market volatility, and any and all world state conditions in your favor*
>
> *The result? Your target will do exactly what you want, when you want, will apologize profusely for ever thinking you were not God Almighty, and will thank you for the opportunity to slave for you for another couple of years.*
>
> *Staff. Retention. Achieved.*
```

slight format change to highlight that this section is from perspective of (and in tone of) counter party adversary

### Wait, am I the sucker?

> If you're bringing it that big paycheck at FAANG or the likes, chances are, you're too deep in that quicksand to get out. Every year you tell yourself and your family "one more year" and "after this project is over" and "just as soon as I get this promotion", but every year the next incentive comes up.

Reviewer: ```suggestion
If you're bringing in that big paycheck at FAANG or the likes, chances are, you're too deep in that quicksand to get out. Every year you tell yourself and your family "one more year" and "after this project is over" and "just as soon as I get this promotion", but every year the next incentive comes up.
```

assumed typo
